src/Native/Windows/build-native.cmd

```diff
@@ -111,6 +111,13 @@ echo See: https://github.com/dotnet/coreclr/blob/master/Documentation/project-do
 exit /b 1
 
 :GenVSSolution
+:: Generate Version file
+if not exist "%__IntermediatesDir%\clrcompression" md "%__IntermediatesDir%\clrcompression"
+msbuild /t:GenerateVersionHeader build.proj /p:NativeVersionHeaderFile="%__IntermediatesDir%\clrcompression\_version.h" /p:GenerateVersionHeader=true
+:: Copy NativeVersion.rc to the Intermediate dir
+if [%TOOLS_DIR%]==[] set TOOLS_DIR=%__binDir%\..\Tools\
+if not exist "%__IntermediatesDir%\clrcompression\NativeVersion.rc" copy "%TOOLS_DIR%NativeVersion.rc" "%__IntermediatesDir%\clrcompression\NativeVersion.rc"
```

Comment:
Are we expecting that we need to for every native project? At a minimum we should move this copy logic into the msbuild target we are calling that generates the header file so we can reduce this to one line per project.